### PR TITLE
libpcap: Update to 1.10.1

### PR DESCRIPTION
**Pull request by ghost**
Update the libpcap package to the latest version 1.10.1.
The version we currently offer is listed as vulnerable at Repology.

### CODE REVIEW

**Comment by AndWac** [comment]
Have you checked that all dependent packages are 64 bit only?

**Comment by ghost** [comment]
I messed up the FMRI first and therefore couldn't see any package that depends on it.
I'll have to re-check everything now,maybe some of them even need to be rebuilt or have newer versions available.
I'll also check if they're all 64bit-only already...

**Comment by ghost** [comment]
There are quite a lot of dependents and some of them are still available in 32bit.
Maybe some of them can be upgraded to 64bit-only later, but for now it's simpler to keep the 32bit compatibility of libpcap.
At least everything still works without rebuilding.

File: components/library/libpcap/libpcap.p5m [hunk 1]
```diff
@@ -11,40 +11,49 @@
...
+file path=usr/include/pcap/socket.h
 file path=usr/include/pcap/usb.h
 file path=usr/include/pcap/vlan.h
+file path=usr/lib/$(MACH64)/libpcap.a
 link path=usr/lib/$(MACH64)/libpcap.so target=libpcap.so.1
 file path=usr/lib/$(MACH64)/libpcap.so.$(COMPONENT_VERSION)
 link path=usr/lib/$(MACH64)/libpcap.so.1 target=libpcap.so.$(COMPONENT_VERSION)
+file path=usr/lib/$(MACH64)/pkgconfig/libpcap.pc
+file path=usr/lib/libpcap.a
 link path=usr/lib/libpcap.so target=libpcap.so.1
```
Comment (AndWac) [inline components/library/libpcap/libpcap.p5m:51]: Here we should drop the static libs.

**Pull request merged**